ios/FacebookPlugin.mm

```diff
@@ -275,11 +275,8 @@ - (FBSDKGameRequestContent *) resolveGameContentFromParams:(NSDictionary *)param
     } else if ([params objectForKey:@"href"]) {
         FBSDKShareLinkContent *content = [[FBSDKShareLinkContent alloc] init];
         content.contentURL = [params objectForKey:@"href"];
-        content.contentTitle = [params objectForKey:@"title"];
-        content.contentDescription = [params objectForKey:@"description"];
         NSString *stringPictureUrl = [params objectForKey:@"picture"];
         NSURL *pictureURL = [NSURL URLWithString:stringPictureUrl];
-        content.imageURL = pictureURL;
```

Comment:
These are deprecated.
https://stackoverflow.com/questions/44733466/facebook-share-in-ios-with-the-new-updated-apis-in-v2-9